### PR TITLE
Update icon list

### PR DESCRIPTION
**Pull request by chris-lawton**
- Remove light/dark iterations of icons as there's only 1 version of them now
- Remove arrow from icon choice list
- Add Magnifying Glass and Code Document to list
- Fix paper plane icon
- Add css for paper plane icon
- Use a percentage to make stroke width on icons consistent

### CODE REVIEW

File: wagtailio/core/choices.py [hunk 1]
```diff
@@ -46,8 +46,7 @@ class SVGIcon(models.TextChoices):
     ROLLBACK = "rollback", "Rollback"
     SLACK = "slack", "Slack"
     SPEECH_BUBBLE = "speech-bubble", "Speech Bubble"
-    SUN_CLOUD_DARK = "sun-cloud-dark", "Sun Cloud Dark"
-    SUN_CLOUD_LIGHT = "sun-cloud-light", "Sun Cloud Light"
+    SUN_CLOUD = "sun-cloud", "Sun Cloud"
```
Comment (chris-lawton) [inline wagtailio/core/choices.py:49]: @engineervix Just FYI i've updated the list of icons after we removed the light/dark versions and added some that were missing

File: wagtailio/core/choices.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 
 class SVGIcon(models.TextChoices):
-    ARROW = "arrow", "Arrow"
```
Comment (William-Blackie) [inline wagtailio/core/choices.py:5]: Question: Did you mean to remove this?

Comment (William-Blackie) [reply wagtailio/core/choices.py:5]: I've just re-read your MR. You did infact mean to remove this :joy:

File: wagtailio/core/migrations/0045_update_icons_list.py [hunk 3]
```diff
@@ -0,0 +1,1259 @@
+# Generated by Django 3.2.12 on 2022-08-19 09:11
```
Comment (William-Blackie) [inline wagtailio/core/migrations/0045_update_icons_list.py:1]: This is a massive nitpick that I feel I should apologise for asking you to rename this slightly. I prefer naming migrations to directly map to a model and what you've changed to generate or the impact. In this case, you've changed the class `SVGIcon` and as it's inheriting from `model.text_choices` I would think semantically the name of `update_svg_icon_choices` as a name is more accurate.

**Review by William-Blackie — APPROVED** [review]
Looks great @chris-lawton, nice work! One very small nitpick which I wouldn't even mind you not doing.

File: wagtailio/core/migrations/0045_update_icons_list.py (hunk 3 above)
Comment (chris-lawton) [reply wagtailio/core/migrations/0045_update_icons_list.py]: Renamed - thanks Will!

**Pull request merged**